install/installer/traf_config_check

```diff
@@ -595,10 +598,40 @@ else
 fi
 }
 
+function setPath {
+
+export PATH=$PATH:$HADOOP_PREFIX/bin:$HADOOP_PREFIX/sbin:$HBASE_HOME/bin
+
+sudo chmod 777 $TRAF_CONFIG
+sed -i '/PATH\=/d' $TRAF_CONFIG
+echo "export PATH=\"$PATH\"" >> $TRAF_CONFIG
+sudo chmod 777 $TRAF_CONFIG
+source $TRAF_CONFIG
+
+}
+
+function setHBaseDistro {
+
+export HBASE_DISTRO="APACHE" 
+
```

Comment:
I agree. This variable is only used with Hadoop version is Apache in sqenvcom.sh. 

A JIRA to improve sqnencom.sh to be more standard and use more of the current variables would be a good idea.